jup-ag/terminal
# [DRAFT] Support market prioritization fee (#55)

Took over from https://github.com/jup-ag/terminal/pull/55

Will be a minor version upgrade, dropping `computeBudgetInstructions`

Things to test
- [ ] onRequestIx should respect the new priority fee multiplier
- [ ] swap should respect the new priority fee multiplier
- [ ] Update default slippage to 0.3, 0.5, 1.0
  - [ ] defaulted to 0.5
- [ ] determine if dropping `computeBudgetInstructions` will affect existing versions
- [ ] Just show wallet icon upon connection
---


* bump jup-ag/react-hook to 6.0.0-alpha.7

* New Priority typing

* CheckIcon component

* Update Terminal to support market prioritization fee

* Update PRIORITY_PRESET array

* Fix onSubmit callback not refresh after prioritizationFeeLamports changed

* UI improvement for SwapSettingModal

* Create FormSettingButton, SettingGearIcon and TransactionPrioritySVG component

* Split Priority, Slippage and General setting to 3 different modal

* Update all modal with respective copy

---------
